app/models/produce.rb

```diff
@@ -1,7 +1,6 @@
 class Produce < ApplicationRecord
-  self.primary_key = 'id'
   has_many :Crops
   validates :name, presence: true, format: { with: /\A[a-zA-Z]*\Z/ } # check if the name has only letters
-  validates :name, :uniqueness => { :case_sensitive => false } # only one produce of a particular name should exist
-  validates :duration, presence: true, format: { with: /\A[0-9]*\Z/ } # duration should only be numbers
+  validates :name, uniqueness: { case_sensitive: false } # only one produce of a particular name should exist
+  validates :type, presence: true, format: { with: /\A(PerennialProduce)|(AnnualProduce)\Z/, message: 'Only PerennialProduce or AnnualProduce is allowed' } # check if the name has only letters
 end
```

Comment:
Want to make sure that I merged this properly

Reply:
Yes, this looks good.